README.md

```diff
@@ -130,12 +130,12 @@ The Julia source code is organized as follows:
 
 ## Terminal, Editors and IDEs
 
-The Julia REPL is quite powerful.  See the section in the manual on
```

Comment:
Regarding this, it's redundant. GitHub displays the line the same way with 1 and 2 spaces here.